.github/workflows/releases.yml

```diff
@@ -0,0 +1,28 @@
+# A GitHub action that notifies the developer
+# changelog repository of any new releases.
+
+name: Notify changelog
+
+on:
+  # Only trigger for a full release,
+  # ignoring pre-releases and drafts
+  release:
+    types:
+      - released
```

Comment:
We typically mark our releases as pre-release until we release to production, when does this action fire?

Reply:
This ignores all the pre-releases, which I think is probably what we want for the public changelog on developer.box.com.

Reply:
I also think it would still be helpful to know specifically what conditions cause this action to fire, per @ConradJChan's original question.

Reply:
So the GitHub API lists this as follows:

https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#release

---

The action that was performed. Can be one of:
- published: a release, pre-release, or draft of a release is published
- unpublished: a release or pre-release is deleted
- created: a draft is saved, or a release or pre-release is published without previously being saved as a draft
-edited: a release, pre-release, or draft release is edited
- deleted: a release, pre-release, or draft release is deleted
- prereleased: a pre-release is created
- released: a release or draft of a release is published, or a pre-release is changed to a release

---

I'd love to know how you currently roll out your releases. Depending on how you roll them out, we can either implement something in this GitHub action, or alternatively we can implement something in the changelog repo to ignore some of your releases. 

Can you clarify for me (@jstoffan @ConradJChan) how you currently roll out releases? I'm also more than happy to jump on a call later this week.

Reply:
We use a custom [release script](https://github.com/box/box-content-preview/blob/master/build/release.sh#L232) to trigger a Github release via [conventional-github-releaser](https://www.npmjs.com/package/conventional-github-releaser). We don't use semver for prereleases, though, so each is cut as a full release that we then modify manually while QA validates the new version.

Happy to chat more offline, feel free to set up some time as needed.

Reply:
While we could create our releases as prereleases, I'm not sure we would want to do that because we would then need to cut a proper release just to trigger this workflow. Thoughts on changing the `types` above to `edit`? That seems like it would fall in line with how we roll out our releases with the manual change from pre-release to release?